templates/users/header.tera

```diff
@@ -11,7 +11,7 @@
     </h1>
 
     {% if is_self %}
-        <a href="edit" class="button inline-block">{{ "Edit your profile" | _ }}</a>
+        <a href="{{ name }}/edit" class="button inline-block">{{ "Edit your profile" | _ }}</a>
```

Comment:
This link won't always be valid: if you are on `/@/my_user_name/` instead of `/@/my_user_name` it will be broken. It's always the same issue with trailing slashes. The solution here would be to have an absolute link, like `/@/{{ user.fqn }}/edit/`.

Reply:
I see the point here, I will test this possibility but I see another point for the future, if you have Plume behind a reverse proxy and in a subdirectory of a bigger web site you have another problem with absolute url i think, or Rocket can manage this case ?

Reply:
I think it would probably require to modify a bit the code (change the Rocket mount point in `main.rs`), but it should be possible.